crates/sui/src/sui_commands.rs

```diff
@@ -63,32 +63,42 @@ const DEFAULT_FAUCET_MIST_AMOUNT: u64 = 200_000_000_000; // 200 SUI
 #[cfg(feature = "indexer")]
 #[derive(Args)]
 pub struct IndexerFeatureArgs {
-    /// Start an indexer with default host and port: 0.0.0.0:9124, or on the port provided.
+    /// Start an indexer with default host and port: 127.0.0.1:9124, or on the port provided.
     /// When providing a specific value, please use the = sign between the flag and value:
     /// `--with-indexer=6124`.
     /// The indexer will be started in writer mode and reader mode.
     #[clap(long,
+            default_value = "9124",
```

Comment:
What's the distinction between `default_value` and `default_missing_value` -- do we need both?

Reply:
Ah I see -- I guess my thinking here was that it shouldn't matter that it was zero if the flag was not passed, because at that point, the port was not going to be used for anything. Combining host and port into one flag addresses this issue anyway, I think.